File: FitoGrano/Functions/mdVariables.vb
Imports DevExpress.LookAndFeel
Imports DevExpress.Utils.Animation

Module mdVariables
    ' модуль для складирования переменных
    Friend iLookAndFeel As New DefaultLookAndFeel                                   ' Стили оформления для программы
    Friend pref_RegistryPath As String = "Software\Microsoft\FitoGrano\"            ' Префикс пути в реестре к настройкам
    Friend pref_ComplexSettings As String = pref_RegistryPath & "ComplexSettings\"  ' Префикс пути в реестре к настройкам комплекса
    Friend pref_UserSettings As String = pref_RegistryPath & "UserSettings\"        ' Префикс пути в реестре к настройкам пользователя
    Friend tmWaitAnimation As New TransitionManager                                 ' Анимация при ожидании
    Friend DecSeporator As String                                                   ' Разделитель десятичной доли заданный в системе пользователя
    ' Friend AddEdit As Integer = 2                                                   ' 2 - Добавить; 3 - Изменить
    Friend EventChangedControl As Boolean = False                                   ' Разрешение на обработку событий элементов управления
    Friend glNewId As String                                                        ' Ид новой записи в базе (глобальная переменная)

    ' Переменные для хранения параметров
    Friend pref_PerformerId As String                                               ' id текущего пользователя
    Friend pref_PerformerName As String                                             ' Полное имя текущего пользователя
    Friend pref_PerformerNameShort As String                                        ' ФИО текущего пользователя
    Friend pref_PerformerLogin As String                                            ' Логин текущего пользователя
    Friend pref_ServerIP As String                                                  ' ip текущего сервера
    Friend pref_DataBaseName As String                                              ' Имя текущей базы данных  
    Friend pref_OrganizationFilial As String                                        ' Текущий филиал  
    Friend pref_ConnectionString As String                                          ' Строка подключения текущего пользователя

    ' Пол
    Public Enum SexType
        Женский = 0
        Мужской = 1
    End Enum
End Module

File: FitoGrano/CS/Functions/mdVariables.cs
// VBConversions Note: VB project level imports
using System.Collections.Generic;
using System;
using System.Linq;
using System.Drawing;
using System.Diagnostics;
using Microsoft.VisualBasic;
using System.Xml.Linq;
using System.Windows.Forms;
// End of VB project level imports

using DevExpress.LookAndFeel;
using DevExpress.Utils.Animation;
using FitoGrano;


namespace FitoGrano
{
	sealed class mdVariables
	{
		// модуль для складирования переменных
		internal static DefaultLookAndFeel iLookAndFeel = new DefaultLookAndFeel(); // Стили оформления для программы
		internal static string pref_RegistryPath = "Software\\Microsoft\\FitoGrano\\"; // Префикс пути в реестре к настройкам
		internal static string pref_ComplexSettings = "Software\\Microsoft\\FitoGrano\\" + "ComplexSettings\\"; // Префикс пути в реестре к настройкам комплекса
		internal static string pref_UserSettings = "Software\\Microsoft\\FitoGrano\\" + "UserSettings\\"; // Префикс пути в реестре к настройкам пользователя
		internal static TransitionManager tmWaitAnimation = new TransitionManager(); // Анимация при ожидании
		internal static string DecSeporator; // Разделитель десятичной доли заданный в системе пользователя
		// Friend AddEdit As Integer = 2                                                   ' 2 - Добавить; 3 - Изменить
		internal static bool EventChangedControl = false; // Разрешение на обработку событий элементов управления
		internal static string glNewId; // Ид новой записи в базе (глобальная переменная)
		
		// Переменные для хранения параметров
		internal static string pref_PerformerId; // id текущего пользователя
		internal static string pref_PerformerName; // Полное имя текущего пользователя
		internal static string pref_PerformerNameShort; // ФИО текущего пользователя
		internal static string pref_PerformerLogin; // Логин текущего пользователя
		internal static string pref_ServerIP; // ip текущего сервера
		internal static string pref_DataBaseName; // Имя текущей базы данных
		internal static string pref_OrganizationFilial; // Текущий филиал
		internal static string pref_ConnectionString; // Строка подключения текущего пользователя
		
		// Пол
		public enum SexType
		{
			Женский = 0,
			Мужской = 1
		}
	}
	
}
